packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx

```diff
@@ -22,11 +22,6 @@ import { VerifyDomainScreen } from './VerifyDomainScreen';
 type DomainListProps = GetDomainsParams & {
   verificationStatus?: OrganizationDomainVerificationStatus;
   enrollmentMode?: OrganizationEnrollmentMode;
-  /**
-   * Enables internal links to navigate to the correct page
-   * based on when this component is used
-   */
-  redirectSubPath: 'organization-settings/domain' | 'domain';
```

Comment:
@octoper Is this ok to be removed?

Reply:
Yeah should be safe to remove that now that everything is inlined.